iOS/_Stream Temp/ChatMessage+Extensions.swift

```diff
@@ -46,15 +46,19 @@ extension Message: Messageable {
         return .passive
     }
     
-    var lastReadAt: Date? {
+    var lastUpdatedAt: Date? {
```

Comment:
Change this back to last read. And just use an if statement for determining which date to show.